packages/ui-components/src/Common/AvatarGroup/index.tsx

```diff
@@ -65,7 +65,14 @@ const AvatarGroup: FC<AvatarGroupProps> = ({
 
       {avatars.length > limit && (
         <span
+          role="button"
```

Comment:
You can already navigate through avatars with tab without any of this. -1 to the change.